python/fastsim/plot_module.py

```diff
@@ -0,0 +1,200 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from fuzzywuzzy import fuzz, process
+
+
+
+available_signals = {
+   'Fuel Cell Output Power Achieved': 'fc_kw_out_ach',
```

Comment:
@bijay079 Can you push your changes to these names? I want to make sure we change "fuel cell" to "fuel converter" wherever it appears